server/data/ui_config.json

```diff
@@ -121,6 +121,6 @@
         "init_version": 2,
```

Comment:
you should also increase the version so it will actually override previous config